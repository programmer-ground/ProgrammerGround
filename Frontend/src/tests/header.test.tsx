import React from 'react';
import { render } from 'react-dom';
import Header from '../components/header';

it('Header Component - snapshot', () => {
	const el = document.createElement('div');
	render(<Header />, el);
	expect(el.innerHTML).toMatchSnapshot();
});
