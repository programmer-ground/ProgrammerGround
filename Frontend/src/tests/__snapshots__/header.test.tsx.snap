// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component - snapshot 1`] = `"<div class=\\"sc-gsTCUz gWDeLb\\"><a href=\\"http://localhost:3000/playground\\"><img class=\\"sc-bdfBwQ eLuqkg\\"></a><div class=\\"sc-dlfnbm cjNpKg\\"><img class=\\"sc-hKgILt\\"><img class=\\"sc-eCssSg\\"><img class=\\"sc-jSgupP\\"></div></div>"`;
